package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "todo_app",
+  "version": "1.0.0",
+  "description": "A guide to help developers onboard to the Button stack by building a simple todo app.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
you need a script here to point postgraphile to your database & allow you to view the graphiql interface we looked at this morning.
The quickstart command here: https://www.graphile.org/postgraphile/introduction/ should be similar to what you need.
If it's working correctly, you should be able to go to the /graphiql endpoint and see the interface.

Reply:
I was running it manually but
have added it onto script at the step 4.